README.md

```diff
@@ -50,6 +50,7 @@ These tools use the Remote Execution API to distribute builds to workers.
 * [Recc](https://gitlab.com/bloomberg/recc)
 * [Reclient](https://github.com/bazelbuild/reclient)
 * [Siso](https://chromium.googlesource.com/infra/infra/+/refs/heads/main/go/src/infra/build/siso/)
+* [trexe](https://gitlab.com/BuildGrid/trexe) (open source)
```

Comment:
Should we remove the `(open source)` part here? because it would imply that the other clients are not open source (which they are).

Reply:
Makes sense, changed.